atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -146,10 +236,26 @@ def get_buildroot(self, worker_metadatas):
         :return: list, containing dicts of partial metadata
         """
         buildroots = []
-        for platform in sorted(worker_metadatas.keys()):
-            for instance in worker_metadatas[platform]['buildroots']:
-                instance['id'] = '{}-{}'.format(platform, instance['id'])
-                buildroots.append(instance)
+
+        if self.source_build:
+            buildroot = get_buildroot(build_id=self.build_id, tasker=self.tasker,
+                                      osbs=self.osbs, rpms=False)
+            buildroot['id'] = '{}-{}'.format(buildroot['container']['arch'], buildroot['id'])
+
+            registry = self.workflow.push_conf.docker_registries[0]
+            build_name = get_unique_images(self.workflow)[0].to_str()
+
+            buildroot['extra']['osbs']['koji'] = {
```

Comment:
Should this be `buildroot['extra']['osbs'] = {` ?

Reply:
nope, this is same as in koji_upload
https://github.com/containerbuildsystem/atomic-reactor/blob/master/atomic_reactor/plugins/post_koji_upload.py#L444